### DISCUSSION TITLE
[RFC FS-1026 Discussion] Allow "params" dictionary as method argument

### DISCUSSION THREAD

**Discussion by neoeinstein** [fsharp/fslang-design#164] [Language and core library RFC discussions]
This is the discussion thread for RFC [FS-1026](https://github.com/fsharp/fslang-design/blob/master/RFCs/FS-1026-allow-params-dictionaries-as-method-arguments.md).

**Comment by rmunn** [comment]
The link in the issue description is (currently) an edit link, which results in anyone who doesn't have write access to this repo seeing a "You need to fork this repository to propose changes" page. Simply replacing the word "edit" with the word "blob" in the URL will fix that.

**Comment by neoeinstein** [comment]
@rmunn Thanks! Fixed.
